### ISSUE TITLE
Detail view between search results and sidebar feels unintuitive

### ISSUE THREAD

**Issue by gravicle** [orta/GIFs#29]

**Comment by fuzzy76** [comment]
Yes, the normal way of structuring this is going from general to more specific in a direction since that is the way we scan for information. So sources, list, detail view makes more sense.

**Comment by orta** [comment]
I agree, but GIFs would crash every time I tried to re-order the views in the split view, thus I shipped it as is.

**Comment by orta** [comment]
Well, guess who just tested if this can work?

**Comment by orta** [comment]
And it doesn't work :dancers:
